src/Entity/User.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\Collection;
+
+class User
+{
+    private int $id;
+
+    private string $password;
+
+    private Collection $roles;
+
+    private string $cnp;
+
+    public string $firstName = '';
+
+    public string $lastName = '';
+
+    private Collection $programs;
```

Comment:
if the entity is called `Programme`, you should be consistent and call this property `programmes`

Reply:
OK